jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/ConcurrentPool.java

```diff
@@ -46,6 +45,7 @@
 @ManagedObject
 public class ConcurrentPool<P> implements Pool<P>, Dumpable
 {
+    static final int MAX_RECOMMENDED_SIZE = 256;
```

Comment:
I feared that question, as the answer is based on a back-of-the-envelope calculation backed by a quick & dirty test.

I'll document that, since this constant isn't public in any way, it might be good enough.